aip-console-tools-core/pom.xml

```diff
@@ -62,6 +62,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>11.0.1</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Please remove after updating the list of patterns in the LogUtils.

Reply:
Yeah! got removed